docs/csharp/language-reference/keywords/volatile.md

```diff
@@ -9,42 +9,46 @@ helpviewer_keywords:
 ms.assetid: 78089bc7-7b38-4cfd-9e49-87ac036af009
 ---
 # volatile (C# Reference)
+
 The `volatile` keyword indicates that a field might be modified by multiple threads that are executing at the same time. Fields that are declared `volatile` are not subject to compiler optimizations that assume access by a single thread. These restrictions ensure that all threads will observe volatile writes performed by any other thread in the order in which they were performed. There is no guarantee of a single total ordering of volatile writes as seen from all threads of execution.  
   
- The `volatile` modifier is usually used for a field that is accessed by multiple threads without using the [lock](../../../csharp/language-reference/keywords/lock-statement.md) statement to serialize access.  
+ The `volatile` modifier is usually used for a field that is accessed by multiple threads without using the [lock](lock-statement.md) statement to serialize access.  
   
  The `volatile` keyword can be applied to fields of these types:  
   
--   Reference types.  
-  
--   Pointer types (in an unsafe context). Note that although the pointer itself can be volatile, the object that it points to cannot. In other words, you cannot declare a "pointer to volatile."  
+- Reference types.  
   
--   Types such as sbyte, byte, short, ushort, int, uint, char, float, and bool.  
+- Pointer types (in an unsafe context). Note that although the pointer itself can be volatile, the object that it points to cannot. In other words, you cannot declare a "pointer to volatile."  
   
--   An enum type with one of the following base types: byte, sbyte, short, ushort, int, or uint.  
+- Simple types such as `sbyte`, `byte`, `short`, `ushort`, `int`, `uint`, `char`, `float`, and `bool`.  
```

Comment:
Maybe here, it's worth to mention `Interlocked` for the `long` and `double` types as those types are supported by `volatile`